tests/php/src/Admin/OptionsMenuTest.php

```diff
@@ -198,6 +198,8 @@ public function get_can_validate_data() {
 	 * @param bool $can_validate
 	 */
 	public function test_enqueue_assets_right_hook_suffix( $can_validate ) {
+		$initial_action_count = did_action( 'amp_register_polyfills' );
```

Comment:
The fact that this is not always initially `0` indicates the test suite is not cleaning itself up always.